pkg/target/manager.go

```diff
@@ -254,7 +258,25 @@ func (m *managerImpl) TargetShoot(ctx context.Context, shootName string) error {
 
 	tb.Init(currentTarget)
 
-	target, err := tb.SetShoot(ctx, shootName).Build()
+	target, err := tb.SetShoot(ctx, shootName).SetShootControlPlane(ctx, m.TargetFlags().ControlPlane()).Build()
```

Comment:
```suggestion
	target, err := tb.SetShoot(ctx, shootName).SetControlPlane(ctx, m.TargetFlags().ControlPlane()).Build()
```

Reply:
Yes, I also struggled with this naming. At some parts I added _Shoot_ to the function names to make it clear that we target Shoot Control Plane, not e.g. the one of the Seed. But for now and to align things I'm also happy to remove it, which I'll do.